docs/sdk/migrations/6.0.0.md

```diff
@@ -0,0 +1,491 @@
+---
+sidebar_position: 3
+---
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating from 5.x.x to 6.x.x
+
+## Breaking Changes & Important Updates
+
+The most significant change in v6 is the combination of `authSig` and `sessionSigs`. In most functions, the client-side generated `authSig` will no longer be accepted as an argument, as seen in commonly used functions like `executeJs` and `pkpSign`. Instead, it will be used to generate `sessionSigs` to authenticate with the Lit nodes.
+
+### Per-Package Changes
+
+#### `@lit-protocol/lit-node-client`
+
+##### executeJs
+
+- v5 - optional use of `authSig` or `sessionSigs`, but either must exist.
+- v6 - `sessionSigs` is strictly required
+
+```tsx
+// v5
+import { AuthSig, SessionSigsMap } from '@lit-protocol/types';
+
+const authSig: AuthSig = '...';
+const sessionSigs: SessionSigsMap = '...';
+
+const res = await litNodeClient.executeJs({
```

Comment:
why aren't we provided an `ipfsId` or `code` to `executeJs`

Reply:
the spread `...` means they parameters are unchanged. The ones being shown are changed.

Reply:
> why aren't we provided an ipfsId or code to executeJs

@joshLong145 I think these code snippets are just demonstrating the changes and don't cover the complete implementation. I replaced the use of `...` with `// rest of your code...` to make this a little clearer